### DISCUSSION TITLE
Thermal option with different chemistries

### DISCUSSION THREAD

**Discussion by hindelou** [pybamm-team/PyBaMM#1997] [Q&A] [answered]
Hello, I'm new to Pybamm and I want to know how to simulate the temperature change for different chemistries, because when I use the thermal option with any of the available chemistries, except Marquis2019, I get an error concerning the absence of some parameters.

**Comment by brosaplanella** [comment, marked as answer]
Hi, you need to choose a suitable parameter set that includes the parameters for the thermal model (e.g. ORegan2021) or amend an existing to add such parameters. You can find more on how to change parameters [on this notebook](https://colab.research.google.com/github/pybamm-team/PyBaMM/blob/main/examples/notebooks/Getting%20Started/Tutorial%204%20-%20Setting%20parameter%20values.ipynb)

**Reply by hindelou** [reply]
Thank you for your helpful reply. 
Another question, please. Can I get the runaway temperature or the max temperature of each chemistry because I didn't find it in the parameter values just like the cutoff voltages?
